Emby.Naming/Video/ExtraResolver.cs

```diff
@@ -80,6 +80,15 @@ private ExtraResult GetExtraInfo(string path, ExtraRule rule)
                     result.Rule = rule;
                 }
             }
+            else if (rule.RuleType == ExtraRuleType.DirectoryName)
+            {
+                var directoryName = Path.GetFileName(Path.GetDirectoryName(path));
+                if (string.Equals(directoryName, rule.Token, StringComparison.OrdinalIgnoreCase))
```

Comment:
I would rather we do a more relaxed matching, like `behind the scenes` == `behind.the.scenes` or something similar

Reply:
I was planning on improving this (and all extras matching) in a future PR :)
231ef226d0272feaed12836dbc933031a09b2afe